### DISCUSSION TITLE
Buttons on footer

### DISCUSSION THREAD

**Discussion by YnievesDotNet** [Power-Components/livewire-powergrid#1385] [Ideas]
How could I add buttons to the table footer? I mean, just like how I put buttons in the header, can I also put them in the footer?

Or make a button a group, something like:

[ Load orders [ Selected | All ] ]

because when I put some buttons the bar use two columns

and i wan put in the footer the massive buttons orders and in the header for selected items

Thanks

**Comment by YnievesDotNet** [comment]
<img width="1200" alt="Captura de pantalla 2024-02-04 a la(s) 1 13 47 p m" src="https://github.com/Power-Components/livewire-powergrid/assets/1451485/7a4fdbdb-7f36-4583-bad6-344fc02ca898">
Thats button can go to the footer, or on another row in headers, separated of other two bottons

**Reply by luanfreitasdev** [reply]
Hello. How are you rendering these actions?

**Reply by luanfreitasdev** [reply]
You can't do this natively. To do this you should use includeViewOnBottom (Header or Footer)

https://livewire-powergrid.com/table/features-setup.html#includeviewonbottom

**Reply by YnievesDotNet** [reply]
thanks, for now are working editing the templates views, and work fine. in the second version I talk here other requirements.

I like this Component... thanks Luan
